### PR TITLE
Improve the quiz extra 2 and test

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

This PR Close y-martinez/PyZombis#18
## Checklist

- [X] Variables, functions and comments are translated to Spanish
- [X] Functions follow underscore notation
- [X] Spell check done & typos fixed
- [X] All python code is PEP8 compliant
- [X] Test coverage with Playwright implemented; locators are Pyhton code
- [X] Reviewers assigned (all peers & at least 1 mentor)

### Test Quiz Extra 2 Exercise 3 bad result
test_quizExtra_3_bad_chromium

### CODE REVIEW

File: _sources/quiz/QuizExtras2.rst [hunk 1]
```diff
@@ -6,87 +6,117 @@ Quiz - Extras 2
...
-   Para isso, a escola pretende gastar EXATAMENTE 93 reais e sabe que a entrada do zoológico custa 5 reais para menores de 12 anos e 7 reais aqueles que têm 12 anos ou mais. |br|
-   Desenvolver um algoritmo que retorne o número máximo de alunos que a escola pode levar para o zoológico considerando todos os valores como inteiros. |br|
-   Seu programa deve ter uma abordagem genérica e não levar em conta peculiaridades nos dados fornecidos |br|
-   Retornar o número total de alunos. |br|
+    Una escuela primaria tiene planeado hacer una visita al zoológico a algunos de sus alumnos. Para esto la escuela solamente puede gastar exactamente ``presupuesto`` unidades y se sabe que la entrada al zoológico cuesta 5 unidades para menores de 12 años y 7 unidades para los que tengan 12 años o mas. Desarrolle la función ``maximo_estudiantes`` que recibe ``presupuesto`` un entero positivo que indica el presupuesto que pretende gasta la escuela. La función debe devolver una tupla ``(p, g)`` con el número máximo pequeños y grandes alumnos que la escuela puede llevar al zoológico considerando todos lo valores como enteros. |br| |br|
```
Comment (angelasofiaremolinagutierrez) [inline _sources/quiz/QuizExtras2.rst:15]:
```suggestion
    Una escuela primaria tiene planeado hacer una visita al zoológico a algunos de sus alumnos. Para esto la escuela solo puede gastar exactamente ``presupuesto`` unidades. Se sabe que la entrada al zoológico cuesta 5 unidades para menores de 12 años y 7 unidades para los que tengan 12 años o más. Desarrolle la función ``maximo_estudiantes`` que recibe ``presupuesto`` un entero positivo que indica el presupuesto que pretende gasta la escuela. La función debe devolver una tupla ``(p, g)`` con el número máximo pequeños y grandes alumnos que la escuela puede llevar al zoológico considerando todos los valores como enteros. |br| |br|
```

File: _sources/quiz/QuizExtras2.rst [hunk 2]
```diff
@@ -6,87 +6,117 @@ Quiz - Extras 2
...
+Ejercicio 3
+-----------
 
-       
+..  activecode:: qExtra_3
+    :nocodelens:
 
-   ====
-   from unittest.gui import TestCaseGui
+    Desarrolle la función ``calcular_pi`` que recibe un parámetro ``n`` positivo mayor a 0. La función debe calcular el valor aproximado de ``pi`` con ``n`` terminos, de acuerdo a la siguiente fórmula. |br|
```
Comment (angelasofiaremolinagutierrez) [inline _sources/quiz/QuizExtras2.rst:88]:
```suggestion
    Desarrolle la función ``calcular_pi`` que recibe un parámetro ``n`` positivo mayor a 0. La función debe calcular el valor aproximado de ``pi`` con ``n`` terminos, de acuerdo con la siguiente fórmula. |br|
```

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
Just small suggestions. Everything else looks ok! :)

**Review by LeoCumpli21 — APPROVED** [review]
Just take a look at what @angelasofiaremolinagutierrez suggested. 

Update: Actually, why don't you change the IDs to **qExtra2_#**, so that the *QuizExtras* (the one that I get to fix) adopt these IDs format (qExtra_#).

**Comment by y-martinez** [comment]
> Just take a look at what @angelasofiaremolinagutierrez suggested.
> 
> Update: Actually, why don't you change the IDs to **qExtra2_#**, so that the _QuizExtras_ (the one that I get to fix) adopt these IDs format (qExtra_#).

Cool! I already changed them 👍🏼

File: tests/test_quizExtra2.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+def test_quizExtra_3_bad(page):
```
Comment (LeoCumpli21) [inline tests/test_quizExtra2.py:1]:
```suggestion
def test_quizExtra2_3_bad(page):
```

**Review by eamanu — APPROVED** [review]

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**